app/lib/validation.js

```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash')
+
+const addEntityIfValid = function (object, params, modelName, name) {
+  try {
+    object[name] = Reflect.construct(modelName, [params]).basicObject()
+  } catch (e) {
+    console.log(`${modelName.name} not added: ${e.message}`)
```

Comment:
If this log will stay in it should use `lib/logger`, this will format the message in a standard way and (experimentally) provide the context/ request ID that caused these models to be made. 

If it's not something that we need to see in every day production it could use the `debug` or `silly` levels (I think debug is used throughout the app so that may be appropriate).